utils/hooks/prepare-commit-msg

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+# This commit message preparattion hook is inspired on the one in ITK.
+# It only works for French locales, since the target tokens to find an
+# appropriate place for the instructions are written in French in this script.
```

Comment:
I am not sure I understand what this means. My computer is completely in English. Will it work?

Reply:
If your git client is in French (have not looked into how this is managed), it will work. If not, you can try adding the lines I mentioned (and executing the setup script again) to check whether that would work.